### PR TITLE
Remove extraneous commented-out code (#138)

### PR DESCRIPTION
**Pull request by sbutkovi**
- Remove extraneous commented-out code (with a focus on removing outdated styling, consistent with issue #138)
- Fix observed typos

I acknowledge that some of the removals I am suggesting may not be desirable (_e.g._, if a comment is intended to stay for later use), so I would appreciate review(s) with that fact in mind - thanks.

### CODE REVIEW

File: src/styles/_variables.scss [hunk 1]
```diff
@@ -41,7 +41,6 @@ $light: #ecf0f1 !default;
 $dark: #7b8a8b !default;
 
 $colors: () !default;
-// stylelint-disable-next-line scss/dollar-variable-default
```
Comment (luxaritas) [inline src/styles/_variables.scss:44]: This is a linter directive, not commented out code

File: src/views/challenges/ChallengeView/components/ChallengeInfoPanel.vue [hunk 2]
```diff
@@ -10,19 +10,6 @@
...
-        {{ $t('challenge-sign-up:subtitle') }}
-      </p>
-      <b-input class="sign-up-container-input" placeholder="Sign up" />
-      <b-button
-        class="sign-up-container-button"
-        variant="primary"
-      >
-        {{ $t('challenge-sign-up:button') }}
-      </b-button>
-    </div> -->
```
Comment (luxaritas) [inline src/views/challenges/ChallengeView/components/ChallengeInfoPanel.vue:25]: Please preserve this for future use

File: src/views/feed/ActivityFeed/ActivityFeed.vue [hunk 3]
```diff
@@ -15,7 +15,6 @@
       <Preloader />
     </div>
     <template #sidebar="{ isInSidebar }">
-      <!--SearchPanel v-if="isInSidebar" :isInSidebar="isInSidebar" /-->
```
Comment (luxaritas) [inline src/views/feed/ActivityFeed/ActivityFeed.vue:18]: Please preserve for future use

File: src/views/labs/LabView/components/LabInfoPanel.vue [hunk 4]
```diff
@@ -18,15 +18,12 @@
...
-    
+
     <template v-if="lab.affiliation">
       <span class="gray-header">{{ $t('lab-info:research-affiliation') }}</span><br />
       <span style="font-weight: bold;">{{ lab.affiliation }}</span><br />
       <br />
     </template>
-    <!-- <span class="gray-header">{{ $t('lab-info:currently-active') }}</span -->
-    <!-- ><br /> -->
-    <!-- <b>{{ $t('lab-info:round-number') }} 4</b> -->
```
Comment (luxaritas) [inline src/views/labs/LabView/components/LabInfoPanel.vue:29]: Please preserve for future use

File: src/views/labs/LabView/components/LabRound.vue [hunk 5]
```diff
@@ -4,13 +4,6 @@
       <h4>
         <b>{{ $t('lab-view:round') }} {{ round.round }}</b>
       </h4>
-      <!--span>
-        <b
-          ><span class="gray-header">Status:</span>&nbsp;{{
-            $t(closed ? 'lab-round:results-posted' : 'lab-round:accepting-submissions')
-          }}</b
-        >
-      </!span-->
```
Comment (luxaritas) [inline src/views/labs/LabView/components/LabRound.vue:13]: Please preserve for future use

File: src/views/players/EditProfile/components/EditPlayerAboutMe.vue [hunk 6]
```diff
@@ -14,24 +14,7 @@
...
-          v-show="!addingSection"
-          variant="secondary"
-        >
-          {{ $t('edit-profile:custom-section-add') }}
-        </b-button>
-        -->
       </div>
-      <!--div class="col-md-4">
-        <EditPlayerFeaturedAchievement />
-      </div-->
```
Comment (luxaritas) [inline src/views/players/EditProfile/components/EditPlayerAboutMe.vue:34]: Please preserve for future use

File: src/views/players/PlayerView/components/PlayerHeaderTopRow.vue [hunk 7]
```diff
@@ -1,14 +1,5 @@
...
   <div class="d-flex">
-    <!-- <b-form-checkbox
-      class="mt-1 mr-3"
-      id="checkbox-1"
-      name="checkbox-1"
-      @change="toggleFollow"
-      v-model="follows"
-      >{{ $t('player-view:following') }}
-    </b-form-checkbox> -->
-
```
Comment (luxaritas) [inline src/views/players/PlayerView/components/PlayerHeaderTopRow.vue:11]: Please preserve for future use

File: src/views/players/PlayerView/components/PlayerHeaderTopRow.vue [hunk 8]
```diff
@@ -39,11 +30,6 @@
       .map((u: UserData) => u.uid)
       .includes(String(this.$vxm.user.uid));
 
-    // toggleFollow() {
-    //   const route = this.follows ? UNFOLLOW_ROUTE : FOLLOW_ROUTE;
-    //   axios.post(route + this.user.uid);
-    // }
-
```
Comment (luxaritas) [inline src/views/players/PlayerView/components/PlayerHeaderTopRow.vue:46]: Please preserve for future use

**Review by luxaritas — CHANGES_REQUESTED** [review]
Thanks! Indeed, there are a few places where we had features commented out for future use.

**Comment by sbutkovi** [comment]
I believe all of the requested changes have now been implemented.

**Review by luxaritas — APPROVED** [review]

**Pull request merged**